x-pack/plugins/cross_cluster_replication/public/app/sections/home/home.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { PureComponent, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiSpacer,
+  EuiPageContentHeader,
+  EuiButton,
+} from '@elastic/eui';
+
+import routing from '../../services/routing';
+import { AutoFollowPatternList } from './auto_follow_pattern_list';
+import { BASE_PATH } from '../../../../common/constants';
+import { SectionUnauthorized } from '../../components';
+
+export class CrossClusterReplicationHomeUI extends PureComponent {
+  static propTypes = {
+    autoFollowPatterns: PropTypes.array,
+  }
+
+  state = {
```

Comment:
The way I'm used to seeing state created in a React component is in the constructor.

Reply:
Like discussed on slack, the constructor is not needed anymore for state initialization:
https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599
@cjcenizal @jen-huang